modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMFormFunctionsServlet.java

```diff
@@ -64,58 +64,53 @@ protected void doGet(
 			HttpServletResponse httpServletResponse)
 		throws IOException, ServletException {
 
-		Map<String, DDMExpressionFunction> ddmExpressionFunctions = null;
+		Map<String, DDMExpressionFunctionFactory> ddmExpressionFunctions = null;
 
-		try {
-			ddmExpressionFunctions = getDDMExpressionFunctions();
+		ddmExpressionFunctions = getDDMExpressionFunctions();
 
-			String languageId = ParamUtil.getString(
-				httpServletRequest, "languageId");
+		String languageId = ParamUtil.getString(
```

Comment:
Since it's used only once and we are not using any String's method there is no need for this variable. Pass the method call directly